src/algebra/polynomial/big_operators.lean

```diff
@@ -5,27 +5,24 @@ Authors: Aaron Anderson, Jalex Stark.
 -/
 import data.polynomial.monic
 import tactic.linarith
-
-open polynomial finset
-
 /-!
-# Polynomials
-
-Lemmas for the interaction between polynomials and ∑ and ∏.
+# Lemmas for the interaction between polynomials and ∑ and ∏.
```

Comment:
I wonder if it's worth a sentence below indicating that these are finset.sum and .prod (with links)